synedrion/src/paillier/encryption.rs

```diff
@@ -414,69 +368,69 @@ mod tests {
     };
     use crate::{
         tools::Secret,
-        uint::{HasWide, SecretSigned, SecretUnsigned},
+        uint::{HasWide, SecretSigned},
     };
 
-    fn mul_mod<T>(lhs: &T, rhs: &SecretSigned<T>, modulus: &NonZero<T>) -> T
+    /// Calculates `val` modulo `modulus`, returning the result in range `[0, modulus)`.
```

Comment:
Document that this is vartime?

Reply:
That's a test function, so I don't think it's necessary.